BP/localization/english/religion/BP_religion_christianity_l_english.yml

```diff
@@ -0,0 +1,147 @@
+ #Marcionism
+ marcionism:0 "Marcionism"
+ marcionism_adj:0 "Marcionist"
+ marcionism_adherent:0 "Marcionist"
+ marcionism_adherent_plural:0 "Marcionists"
+ marcionism_desc:0 "Marcionism was an Early Christian dualist belief system that originated in the teachings of Marcion of Sinope at Rome around the year 144 AD. Marcion believed Jesus Christ was the savior sent by God, and Paul of Tarsus was his chief apostle, but he rejected the Hebrew Bible and the God of Israel. Marcionists believed that the wrathful Hebrew God was a separate and lower entity than the all-forgiving God of the New Testament. This belief was in some ways similar to Gnostic Christian theology. Notably, both are dualistic, that is, they posit opposing gods, forces, or principles: one higher, spiritual, and "good", and the other lower, material, and "evil" (compare Manichaeism), in contrast to the orthodox Christian view that "evil" has no independent existence, but is a privation or lack of "good", a view shared by the Jewish theologian Moses Maimonides."
+ #Donatism
+ donatism:0 "Donatism"
+ donatism_adj:0 "Donatist"
+ donatism_adherent:0 "Donatist"
+ donatism_adherent_plural:0 "Donatists"
+ donatism_desc:0 "Donatists were rigorists, holding that the church must be a church of "saints," not "sinners," and that sacraments, such as baptism, administered by Traditores ("traitors", sinful, traitorous clergy) were invalid. Probably in 311 AD, a new bishop of Carthage, Caecilian, was consecrated by someone who had allegedly been a Traditor, Felix of Aptungi, his opponents consecrated a short-lived rival, who was succeeded by Donatus, after whom the schism was named. In 313, a commission appointed by Pope Miltiades fought against the Donatists, but they continued to exist, viewing themselves, and not the Nicene Church, as the "true Church," the only one with "valid sacraments""
+ #Audianism
+ audianism:0 "Audianism"
+ audianism_adj: "Audianistic"
+ audianism_adherent:0 "Audianist"
+ audianism_adherent_plural:0 "Audianists"
+ audianism_desc:0 "Belief named after Audius which believed that God has human form and that one ought to celebrate Jesus' death during the Jewish Passover."
+ #Montanism
+ montanism:0 "Montanism"
+ montanism_adj:0 "Montanist"
+ montanism_adherent:0 "Montanist"
+ montanism_adherent_plural:0 "Montanists"
+ montanism_desc:0 "Montanism was an early Christian movement of the late 2nd century, later referred to by the name of its founder, Montanus. It is a prophetic movement that calls for a reliance on the spontaneity of the Holy Spirit and a more conservative personal ethic."
+ #Georgian Orthodox
+ georgian_orthodox:0 "Kartvelian Orthodoxy"
+ georgian_orthodox_adj:0 "Kartvelian"
+ georgian_orthodox_adherent:0 "Kartvelian"
+ georgian_orthodox_adherent_plural:0 "Kartvelians"
+ georgian_orthodox_desc:0 "The Georgian Orthodox Church is the head church of the Georgian people, founded by the Kingdom of Iberia."
+ #Syrian Orthodox
```

Comment:
This is a religion you decided to add, but I'm not sure a separate "Syriac Orthodox" branch is necessary, since the conflicts are already present through the Monophysites and Miaphysites.

Reply:
Al-Harith ibn Jabalah (528) was a major Ghassanid patron of the Syriac Orthodox Church, which is the only really specific information I've found on Ghassanid religiosity. I haven't found too many sources that give much detail about the actual details of the Monophysite faith of the Ghassanids; I'm suspecting that they may have nominally fallen under the Antiochene See, or one of the competing claimants for the See, given the confusion between Monophsyites and Miaphystism/Severians.

On the other hand, you might have resources that better outline the specifics in better detail. I can be replaced with a monophysite faith with very little trouble.

Reply:
For completeness:  I'll also note some specific Christian faiths from WTWSMS were not included on this pull request and instead modeled as Christian religious doctrines:
Adoptionist, Antinomian, Apollinarian,  Binitarian, Eutychian, Melchisedechian, Monarchian, Monophsyite, Monothelite, Pelagian, Pneumathomachian, Total Depravity, 
And some that weren't added don't that have specific doctrines/tenets associated with them:
Patripassian, Aquileian, Fraticelli, Psilantrhopist

Reply:
> Al-Harith ibn Jabalah (528) was a major Ghassanid patron of the Syriac Orthodox Church, which is the only really specific information I've found on Ghassanid religiosity. I haven't found too many sources that give much detail about the actual details of the Monophysite faith of the Ghassanids; I'm suspecting that they may have nominally fallen under the Antiochene See, or one of the competing claimants for the See, given the confusion between Monophsyites and Miaphystism/Severians.

On the distinction between Miaphysites and Monophysites with regards to the Ghassanids:

> The Ghassanids are Monophysite in the _The Oxford Dictionary of Byzantium_, but Miaphysite in _The Oxford Dictionary of the Middle Ages_. In all precise references they are Monophysite but in more general descriptions Miaphysite.
> Why the confusion? Because outside the specific literature Miaphysite is being used as a more general term encompassing both the more moderate Miaphysites and the more radical Monophysites.
> By the way, "Miaphysite" does not exist as an entry in the Oxford Dictionaries, instead they speak of moderate and extreme monophysites.

> On the other hand, you might have resources that better outline the specifics in better detail. I can be replaced with a monophysite faith with very little trouble. With a little bit of event writing, the actual Christology of the Syriac Orthodox Branch can start as Monophysite and can change if they loose Ghassanid patronage.

To be clear, I do not think there should be a Syriac Orthodox religion at start. I think it is sufficient to have Nicene, Miaphysite and Monophysite faiths, which then represent the different christological tendencies, more moderate or more radical. The label "Syriac Orthodox" is more confusing than anything else in 476, since it describes a cultural distinction rather than a theological and doctrinal one. The later "Syriac" or "Jacobite" Church emerged from Jacob Baradaeus, which was not even born at start. Yet there were still Miaphysites and Monophysites before Baradaeus, since the one nature belief had been condemned by the Council of Chalcedon.